services/generateSwaggerService.js

```diff
@@ -0,0 +1,182 @@
+const requireAll = require('require-all'),
+  path = require('path'),
+  _ = require('lodash'),
+  config = require('../config'),
+  transactionModel = require('../models/transactionModel'),
+  accountModel = require('../models/accountModel'),
+  contract = require('truffle-contract'),
+  fs = require('fs');
+
+let contracts = {};
+
+let contractsPath = path.join(__dirname, '../../../node_modules', 'chronobank-smart-contracts/build/contracts');
```

Comment:
according to our conversation yesterday - please, add chronobank-smart-contracts as dependency to rest module, and replace this ugly construction to normal require (via require-all) with filter. Smth like:
contracts = requireAll({
dirname: 'node_modules/chronobank-smart-contracts/build/contracts',
filter: /(^((ChronoBankPlatformEmitter)|(?!(Emitter|Interface)).)*).json$/,
resolve: Contract => contract(Contract)
});
Or if you find a better solution - then go ahead